### PR TITLE
Research and add extra benchmarks

### LINKED ISSUE #12: Research and add extra benchmarks
https://owasp.org/www-project-top-ten/
https://www.autosar.org/fileadmin/user_upload/standards/adaptive/18-10/AUTOSAR_RS_CPP14Guidelines.pdf
https://cwe.mitre.org
ALL cert sei coding standards: https://wiki.sei.cmu.edu/confluence/display/c/SEI+CERT+C+Coding+Standard

### PR DESCRIPTION
**Pull request by Cheshiriks**
added extra benchmarks

Close #12

### CODE REVIEW

File: benchmarks/cwe_mitre.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    # benchmark name
+    name = 'CWE'
+    # select the most applicable category from the list provided by this repo
+    category = 'CODING_STANDARD'
+    # mark your benchmark with custom tags (your own custom tags OR use existing from other projects)
+    tags = ['portability', 'security', 'safety']
+
+[info]
+    # programming language
+    language = 'language independent'
```
Comment (petertrr) [inline benchmarks/cwe_mitre.toml]: It's usually called `language agnostic`, I think

Comment (Cheshiriks) [reply benchmarks/cwe_mitre.toml]: Ok. Thank you

File: benchmarks/cwe_mitre.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    # benchmark name
+    name = 'CWE'
+    # select the most applicable category from the list provided by this repo
+    category = 'CODING_STANDARD'
+    # mark your benchmark with custom tags (your own custom tags OR use existing from other projects)
+    tags = ['portability', 'security', 'safety']
+
+[info]
+    # programming language
+    language = 'language agnostic'
```
Comment (orchestr7) [inline benchmarks/cwe_mitre.toml:11]: I you sure, I thought that they have some different packages with different languages

**Review by orchestr7 — APPROVED** [review]

File: benchmarks/cwe_mitre.toml (hunk 2 above)
Comment (Cheshiriks) [reply benchmarks/cwe_mitre.toml:11]: CWE is a general list of information security rules. Most of the rules are a general safety recommendation. But there are also a small number of rules aimed at recommendations specifically for the C, C++. Java and PHP languages.

**Review by orchestr7 — APPROVED** [review]

**Pull request merged**